### PR TITLE
Update page in Connect AWS Gateway “cloud-administration-operation”

### PR DESCRIPTION
**Pull request by jcollins-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by jcollins-axway** [comment]
Jira: APIGOV-16135

**Comment by jcollins-axway** [comment]
@kflan-io @sbolosan @dfeldick @vivekschauhan - please review content

File: content/en/docs/central/connect-aws-gateway/cloud-administration-operation.md [hunk 1]
```diff
@@ -88,49 +88,60 @@ Upload all of these resources to an S3 bucket, within the target region. Take no
...
 
 | Parameter Name                | Description                                                                                                                               | Default Value                   | Mode       |
 | ----------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------- | ---------- |
-| APIGWCWRoleSetup              | If set to true, the IAM Role for the API Gateway Service to write logs to CloudWatch will be created                                       | true                            | both       |
-| ConfigServiceSetup            | If set to true, the IAM Role for the Config Service will be created                                                                        | true                            | continuous |
+| AgentResourcesBucket          | The S3 bucket that has the resources needed for deploying this stack, lambda and nested stack templates                                   |                                 | both       |
```
Comment (sbolosan) [inline content/en/docs/central/connect-aws-gateway/cloud-administration-operation.md:121]: ...for deploying this stack, lambda, and nested stack templates 

missing "," (comma) between lamba and (the omega mu =)).

**Review by sbolosan — APPROVED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes, @lbadenhop, please proceed.

**Review by kflan-io — APPROVED** [review]

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**